receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -260,3 +356,64 @@ func TestTranslateV2(t *testing.T) {
 		})
 	}
 }
+
+func TestAddCounterDatapoints(t *testing.T) {
```

Comment:
Nice test, you're making sure everything you wanted is there. I wonder how this test is different from the one above though, do you have any preferences?